internal/framework/flex/autoflex.go

```diff
@@ -74,7 +69,12 @@ var (
 	plural = pluralize.NewClient()
 )
 
-func findFieldFuzzy(ctx context.Context, fieldNameFrom string, typeFrom reflect.Type, typeTo reflect.Type, flexer autoFlexer) (reflect.StructField, bool) {
+type fuzzyFieldFinder struct {
+	prefixRecursionDepth int
+	suffixRecursionDepth int
+}
```

Comment:
Perhaps for a future refactor, but a `newFuzzyFieldFinder` function with functional options could replace initializing the struct inline (`(&fuzzyFieldFinder{}).findField`). 

```go
func newFuzzyFieldFinder(optFn ...finderOptFn) *fuzzyFieldFinder {
        // handle ops here for prefix/suffix recursion depth
}
```